### ISSUE TITLE
Proxyman appears to be messing with body even if no modification is made

### ISSUE THREAD

**Issue by triwav** [ProxymanApp/Proxyman#2123]
## Description
I am having an issue that a request I am trying to modify the url of with the scripting tool is modifying the body in some way. My best guess is that because the code is trying to convert the body to be compatible with the Javascript code it is causing the issue. I do know a request I send in Proxyman has a Content-Length: 3590 and the same endpoint hit with Charles is much smaller at Content-Length: 2207 which would support the theory that there is some conversion going on that isn't quite right. Is there some way to tell it to just use the original body not the modified version? I'm happy to share Charles and Proxyman session files privately

## Steps to Reproduce

1. Setup the scripting tool in Proxyman that matches the url
2. Make request
3. Request fails in Proxyman but succeeds in Charles

## Current Behavior

Seems to mess with the passed in body

## Expected Behavior

Should not mess with passed in body

## Environment

- App version: Proxyman 5.7.0
- macOS version: 14.4.1

**Comment by NghiaTranUIT** [comment]
You're right @triwav The script will modify your request/response when parsing to JavascriptCore 

> Is there some way to tell it to just use the original body not the modified version?
If you use Breakpoint or Map Local, this won't happen.

> I'm happy to share Charles and Proxyman session files privately

Thanks,. please sent to me at nghia@proxyman.io 

I'm happy to investigate it

**Comment by triwav** [comment]
Sent to your email. Thank you.

**Comment by NghiaTranUIT** [comment]
Fixed: https://download.proxyman.io/beta/Proxyman_5.7.0_Fix_scripting_with_text_parser.dmg
